vpr/src/noc/noc_storage.h

```diff
@@ -271,6 +271,26 @@ class NocStorage {
     void set_noc_router_latency(double router_latency);
 
     // general utiliy functions
+    /**
+     * @brief The link is indentified by going through the outoging links of the
```

Comment:
Start with what you do (removes a link between two specified physical routers).

Reply:
Update the link removal code to also change the data for that link (in the link vector) to invalid values, e.g. source = sink = RouterID::Invalid()

Reply:
done